fri/src/prover/mod.rs

```diff
@@ -209,14 +228,23 @@ where
         // projection to reduce the degree of evaluations by N
         let alpha = channel.draw_fri_alpha();
         *evaluations = apply_drp(&transposed_evaluations, self.domain_offset(), alpha);
-
         self.layers.push(FriLayer {
             tree: evaluation_tree,
             evaluations: flatten_vector_elements(transposed_evaluations),
             _base_field: PhantomData,
         });
     }
 
+    /// Builds a last FRI layer by hashing sequentially `evaluations`.
+    fn build_last_layer<const N: usize>(&mut self, channel: &mut C, evaluations: &mut [E]) {
```

Comment:
It doesn't look like we use `N` type parameter here for anything - do we need it?

Reply:
We don't need it since we are not building trees anymore. Fixed it!